ui/imports/shared/controls/DisabledTooltipButton.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.14
+
+import StatusQ.Controls 0.1
+
+Item {
```

Comment:
:police_officer: what are you trying to do here? :D

Reply:
raised https://github.com/status-im/status-desktop/issues/10185 to handle updating the StatusBaseButton as it will impact all usages of this base control